coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,172 @@
+exts = {'.w': 'C',
+        '.cs': 'C#',
+        '.csx': 'C#',
+        '.c': 'C++',
+        '.c++': 'C++',
+        '.cpp': 'C++',
+        '.cxx': 'C++',
+        '.h': 'C++',
```

Comment:
this is kind of difficult, many people use `.h` for C and C++. not sure what to take now though...